src/__snapshots__/index.test.js.snap

```diff
@@ -203,11 +203,9 @@ exports[`should be able to compile list 1`] = `
 
 exports[`should be able to compile list with html tag 1`] = `
```

Comment:
OK, @stof If we remove this testcase, and upgrade, then release as a new major version, I'm ok with that.